app/index.js

```diff
@@ -405,6 +410,12 @@ function handleAppReady() {
     });
   }
 
+  // Initialize MQTT client
+  if (config.mqtt && config.mqtt.enabled) {
```

Comment:
Use the 'modern' way. More concise

```suggestion
  if (config.mqtt?.enabled) {
```